modules/http/src/main/java/org/mule/module/http/internal/request/grizzly/IOStrategyTransportCustomizer.java

```diff
@@ -40,7 +40,7 @@ public IOStrategyTransportCustomizer(String threadNamePrefix, Integer maxWorkerP
     @Override
     public void customize(TCPNIOTransport transport, FilterChainBuilder filterChainBuilder)
     {
-        transport.setIOStrategy(FlowWorkManagerIOStrategy.getInstance());
+        transport.setIOStrategy(WorkerThreadIOStrategy.getInstance());
```

Comment:
The pool in configured right under this line using Grizzly defaults, just like selectors. They do twice the cores count. That hand off is within the AsyncHandler for the NB send() and not necessary for the other methods because they are blocking.